packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -123,7 +123,7 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
         <FocusZone>
           <table
             className={ css('ms-DatePicker-table', styles.table) }
-            role='grid'
+            role='application'
```

Comment:
yes, it'll force the screen reader to change modes. why not just properly structure the table and let users use their accelerators?